### PR TITLE
Remove auth_service from public iiif manifests

### PR DESCRIPTION
**Pull request by masaball**

### CODE REVIEW

**Comment by masaball** [comment]
I'm not sure what is going on. The tests are passing in my local test environment. Any ideas @cjcolvar?

File: app/models/iiif_canvas_presenter.rb [hunk 1]
```diff
@@ -55,13 +55,7 @@ def video_content
...
     def video_display_content(quality)
       IIIFManifest::V3::DisplayContent.new(Rails.application.routes.url_helpers.hls_manifest_master_file_url(master_file.id, quality: quality),
-                                           label: quality,
-                                           width: master_file.width.to_i,
-                                           height: master_file.height.to_i,
-                                           duration: stream_info[:duration],
-                                           type: 'Video',
-                                           format: 'application/x-mpegURL',
-                                           auth_service: auth_service(quality))
+                                           manifest_attributes(quality, 'Video'))
```
Comment (cjcolvar) [inline app/models/iiif_canvas_presenter.rb:58]:
```suggestion
                                           **manifest_attributes(quality, 'Video'))
```

File: app/models/iiif_canvas_presenter.rb [hunk 2]
```diff
@@ -70,11 +64,7 @@ def audio_content
 
     def audio_display_content(quality)
       IIIFManifest::V3::DisplayContent.new(Rails.application.routes.url_helpers.hls_manifest_master_file_url(master_file.id, quality: quality),
-                                           label: quality,
-                                           duration: stream_info[:duration],
-                                           type: 'Sound',
-                                           format: 'application/x-mpegURL',
-                                           auth_service: auth_service(quality))
+                                           manifest_attributes(quality, 'Sound'))
```
Comment (cjcolvar) [inline app/models/iiif_canvas_presenter.rb:67]:
```suggestion
                                           **manifest_attributes(quality, 'Sound'))
```

**Review by cjcolvar — COMMENTED** [review]
I think the change I made in `iiif_manifest` coupled with the encapsulating of the keyword arguments broke things.  The disconnect between your local system and circleci either has to do the version of ruby or of the `iiif_manifest` gem.  The tests passed on circleci  when I made these changes to convert the hash returned by `manifest_attributes` to keyword arguments.

**Pull request merged**